### ISSUE TITLE
Help loss go to infinite

### ISSUE THREAD

**Issue by Enigmo109** [Neerajj9/Text-Detection-using-Yolo-Algorithm-in-keras-tensorflow#9]
Hello,
I'm trying to fit the model with dataset that u linked on readme. During my test loss value starts at 200 - 800 (depend on batch size and lr) and go to infinite in 2-4 steps.

Hope to receive soon a feedback,
best regards

**Comment by Owaiskhan9654** [comment]
Check out this closed issue @Enigmo109 

https://github.com/Neerajj9/Text-Detection-using-Yolo-Algorithm-in-keras-tensorflow/issues/3#issuecomment-594271867
